src/modules/datepicker/classes/calendar-mappings.ts

```diff
@@ -69,7 +69,7 @@ export class DatetimeMappings extends CalendarMappings {
             [CalendarViewType.Month, CalendarViewType.Year],
             [CalendarViewType.Date, CalendarViewType.Month],
             [CalendarViewType.Hour, CalendarViewType.Date],
-            [CalendarViewType.Minute, CalendarViewType.Date]
+            [CalendarViewType.Minute, CalendarViewType.Hour]
```

Comment:
Actually, let's keep this change. Because, in the Calendar module (that this is based on) the left right arrows go across days when looking at minutes, which fits the old behavior. However, the datepicker allows navigation across hours, so it's inconsistent now.